website/content/en/search.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Search Results
```

Comment:
Is this file necessary? How are we handling search in this PR? 

While I think it'll be useful to include a way for users to search the docs for the content they're looking for, I'm inclined to defer that work to a separate PR as this is PR is getting quite lengthy to begin with.

Reply:
this is merely the search template, i'll remove it.

I'm planning on using lunr.js, as it's simple and integrated with hugo.